### PR TITLE
Created/Update Components for Make a Post

### PR DESCRIPTION
**Pull request by Will-Hsu**
Alert: customized alert component for Homehub.
CheckBoxGroup: customized checkbox component for Homehub.
TextArea: has a placeholder field that is grey out in addition to pre-filled content.
Tooltip: background-color responsive to the boolean prop isSingleLine.

All 4 components' storybooks are implemented as well.

### CODE REVIEW

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Yizong98) [inline FrontEnd/src/components/basics/Alert/Alert.module.scss:32]: rem

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Yizong98) [inline FrontEnd/src/components/basics/Alert/Alert.module.scss:34]: why are we mixing em and rem here?

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Yizong98) [inline FrontEnd/src/components/basics/Alert/Alert.module.scss:76]: 1.5 rem is default, no need to specify

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Yizong98) [inline FrontEnd/src/components/basics/Alert/Alert.module.scss:68]: same here

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Yizong98) [inline FrontEnd/src/components/basics/Alert/Alert.module.scss:60]: same here

File: FrontEnd/src/components/basics/Alert/Alert.tsx
Comment (Yizong98) [inline FrontEnd/src/components/basics/Alert/Alert.tsx:16]: it is recommended to use an object: e.g. const IconMapping = {'success':alertIcons.Check, ..... }

File: FrontEnd/src/components/basics/Tooltip/Tooltip.module.scss
Comment (Yizong98) [inline FrontEnd/src/components/basics/Tooltip/Tooltip.module.scss:8]: same here, it needs to be rem

**Review by Yizong98 — CHANGES_REQUESTED** [review]
Extremely good job! Well done. Just need some minor fixes before I give you the approval. Thanls

File: FrontEnd/src/components/basics/Tooltip/Tooltip.module.scss
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Tooltip/Tooltip.module.scss]: fixed

File: FrontEnd/src/components/basics/Alert/Alert.tsx
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Alert/Alert.tsx]: fixed

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Alert/Alert.module.scss]: fixed

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Alert/Alert.module.scss]: fixed

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Alert/Alert.module.scss]: fixed

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Alert/Alert.module.scss]: fixed

File: FrontEnd/src/components/basics/Alert/Alert.module.scss
Comment (Will-Hsu) [reply FrontEnd/src/components/basics/Alert/Alert.module.scss]: fixed

**Pull request merged**
